### PR TITLE
Re: Link to official legislation site from case info tab

### PR DESCRIPTION
**Pull request by gwkim8107**
This PR is for "Link to official legislation site from case info tab #202".

### CODE REVIEW

File: src/components/SingleCaseContainer.js [hunk 1]
```diff
@@ -188,8 +188,28 @@ const SingleCase = ({ id }) => {
                         <tbody className="">
                           {data.legislationToCases.map((obj, i) => (
                             <tr className="" key={`legislation-reference-${i}`}>
-                              <td className="">{obj.legislation.title}</td>
-                              <td className="">{obj.section}</td>
+                              <td className="">
```
Comment (williamparry) [inline src/components/SingleCaseContainer.js]: If there's no className can you please delete the property?

File: src/components/SingleCaseContainer.js [hunk 2]
```diff
@@ -188,8 +188,28 @@ const SingleCase = ({ id }) => {
...
+                                <a
```
Comment (williamparry) [inline src/components/SingleCaseContainer.js]: Can you please use prettier

File: src/components/SingleCaseContainer.js (hunk 1 above)
Comment (gwkim8107) [reply src/components/SingleCaseContainer.js]: Sure, those empty className properties have deleted in the file.

File: src/components/SingleCaseContainer.js (hunk 2 above)
Comment (gwkim8107) [reply src/components/SingleCaseContainer.js]: Code prettier has applied again.

**Pull request merged**